src/qute/commands/registerCommands.ts

```diff
@@ -17,6 +19,11 @@ export function registerVSCodeQuteCommands(context: ExtensionContext) {
   registerJavaDefinitionCommand(context);
   registerConfigurationUpdateCommand(context);
   registerQuteValidationToggleCommand(context);
+  context.subscriptions.push(
```

Comment:
To fix the issue from @fbricon, please add

```ts
// When extension is started, loop for each text documents which are opened to update their language ID.
  workspace.textDocuments.forEach(document => {
    updateQuteLanguageId(context, document, false);
  });
```

exactly like application.properties file.

Reply:
This does the same thing as an active text editor listener, so I believe we can leave out the text editor listener and go with this (since this is only an edge case on Quarkus extension start)